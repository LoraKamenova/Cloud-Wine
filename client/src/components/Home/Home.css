.home-wrapper {
    display: flex;
    /*flex-direction: column;*/
    /*align-items: center;*/
    /*background-color: rgba(0, 0, 0, 0.65);*/
    /*color: white;*/
    width: 60%;
    height: 70%;
    margin: auto;
    /*padding: 1px;*/
    border: 15px solid rgba(0, 0, 0, 0.65);;
}

.home-content-container {
    display: flex;
    flex-direction: column;
    align-items: center;
    background-color: rgba(0, 0, 0, 0.65);
    color: white;
    width: 97%;
    height: 97%;
    margin: auto;
    padding: 0 80px 80px 80px;
}

.home-page-subheading {
    position: relative;
    bottom: -50px;
    font-family: 'Caveat', cursive;
    font-size: 40px ;
    text-align: center;
    font-weight: 400;
    color: #FCB269;
}

.home-page-heading {
    font-family: 'Caveat', cursive;
    font-size: 80px ;
    text-align: center;
    font-weight: 400;
    letter-spacing: 10px;
    margin: 20px;
    color: #FCB269;
}

.home-page-paragraph {
    text-align: center;
}
