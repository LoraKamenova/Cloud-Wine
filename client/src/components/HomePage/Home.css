.home-section {
    display: flex;
    flex-grow: 1;
    justify-content: center;
}

.home-content-wrapper {
    display: flex;
    width: 60%;
    margin: auto;
    padding: 10px;
    border: 15px solid rgba(0, 0, 0, 0.65);;
}

.home-content-container {
    display: flex;
    flex-direction: column;
    align-items: center;
    width: 100%;
    color: white;
    background-color: rgba(0, 0, 0, 0.65);
    padding: 0 80px 80px 80px;
}

.home-page-subheading {
    position: relative;
    bottom: -50px;
    text-align: center;
    font-family: 'Caveat', cursive;
    font-size: 40px;
    font-weight: 400;
    color: #FCB269;
}

.home-page-heading {
    text-align: center;
    font-family: 'Caveat', cursive;
    font-size: 80px;
    font-weight: 400;
    color: #FCB269;
    letter-spacing: 10px;
    margin: 20px;
}

.home-page-paragraph {
    text-align: center;
    line-height: 1.1;
}
