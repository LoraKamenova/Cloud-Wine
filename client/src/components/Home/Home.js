import './Home.css'
import Header from "../Header/Header";

const Home = () => {
    return (
        <main className="home-page">
            <Header />
        </main>
    );
};

export default Home;
