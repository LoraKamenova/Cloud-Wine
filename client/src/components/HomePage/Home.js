import './Home.css'

import {Component} from 'react';

class Home extends Component {
    constructor(props) {
        super(props)
    }

    render() {
        return (
            <section className="home-section">
                <div className="home-content-wrapper">
                    <div className="home-content-container">
                        <div className="heading-wrapper">
                            <h3 className="home-page-subheading">Добре дошли във</h3>
                            <h1 className="home-page-heading">Винописи</h1>
                        </div>
                        <p className="home-page-paragraph">
                            Lorem ipsum dolor sit amet, consectetur adipiscing elit.
                            Phasellus pulvinar tellus nec ante volutpat, vel commodo nisl aliquam. Maecenas nec
                            malesuada risus, ut maximus libero. Aliquam bibendum sollicitudin nibh, ac vehicula leo
                            maximus vitae. Aenean vitae metus luctus, molestie leo ac, ullamcorper justo. Aliquam
                            bibendum
                            sollicitudin nibh, ac vehicula leo
                            maximus vitae. Aenean vitae metus luctus, molestie leo ac, ullamcorper justo.
                        </p>
                    </div>
                </div>
            </section>
        )
    }
}

export default Home;
