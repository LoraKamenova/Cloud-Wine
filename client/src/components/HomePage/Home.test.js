import { BrowserRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import Home from './Home';

describe('DictionaryEntry Component', () => {

    it("renders", () => {
        const { asFragment } = render(<Home />);
        expect(asFragment()).toMatchSnapshot();
    });

});
