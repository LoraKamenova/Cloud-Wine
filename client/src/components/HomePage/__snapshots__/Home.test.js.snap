// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DictionaryEntry Component renders 1`] = `
<DocumentFragment>
  <section
    class="home-section"
  >
    <div
      class="home-content-wrapper"
    >
      <div
        class="home-content-container"
      >
        <div
          class="heading-wrapper"
        >
          <h3
            class="home-page-subheading"
          >
            Добре дошли във
          </h3>
          <h1
            class="home-page-heading"
          >
            Винописи
          </h1>
        </div>
        <p
          class="home-page-paragraph"
        >
          Това е споделено пространство за любителите на винения туризъм. Тук можете да намерите идеи за вашата следваща винена дестинация в България, а след това да ни разкажете за вашите преживявания и впечатления. Имаме и внимателно селектиран винен лист с български вина, заслужаващи вашето внимание, а нашият винен речник винаги е на разположение за обогатяване на вашата обща култура на тема вино.
        </p>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
